docs/manual/scala/guide/cluster/DomainModelling.md

```diff
@@ -330,7 +332,7 @@ abstract class ShoppingCartApplication(context: LagomApplicationContext)
 
 ### Getting instances of the Aggregate Entity
 
-To access instances of the Aggregate (which may be running locally or remotely on the cluster), you should inject the `ClusterSharding` on your service can instantiate an `EntityRef` using the method `entityRefFor`.
+To access instances of the Aggregate (which may be running locally or remotely on the cluster), you should inject the `ClusterSharding` on your service. You can then instantiate an `EntityRef` using the method `entityRefFor`. In our case, the `EntityRef` is typed to only accept `ShoppingCartCommand`s.
```

Comment:
It was missing something on this sentence. 

Moreover, the sentence "In our case, the `EntityRef` is typed to only accept `ShoppingCartCommand`s." was on a paragraph below, but it was looking orphan and out of context. Moving here looks better because we are already talking about `EntityRef`